src/uct/sm/mm/base/mm_iface.c

```diff
@@ -106,22 +106,36 @@ uct_mm_iface_query_tl_devices(uct_md_h md,
 
 static int
 uct_mm_iface_is_reachable(const uct_iface_h tl_iface,
-                          const uct_device_addr_t *dev_addr,
-                          const uct_iface_addr_t *tl_iface_addr)
+                          const uct_iface_is_reachable_params_t *params)
```

Comment:
can we inline uct_mm_iface_is_reachable code in uct_mm_iface_is_reachable_v2?

Reply:
fixed